NAMESPACE

```diff
@@ -89,6 +89,7 @@ S3method(repair_draws,draws_array)
 S3method(repair_draws,draws_df)
 S3method(repair_draws,draws_list)
 S3method(repair_draws,draws_matrix)
+S3method(resample_draws,draws)
```

Comment:
Do we only want this methods to work with draws objects? Not draws_df, draws_list etc?

Reply:
`draws` is the parent class of all the formats so a `.draws` method will apply to all formats.